jersey/jsonp/pom.xml

```diff
@@ -28,6 +28,10 @@
     <artifactId>helidon-jersey-media-jsonp</artifactId>
     <name>Helidon Jersey Media JSON-P</name>
 
+    <properties>
+        <maven.javadoc.skip>true</maven.javadoc.skip>
```

Comment:
This should probably not be skipped.

Reply:
These modules have  no source to document. They already create empty javadoc jars. They are just bundle modules. I have verified javadoc won't run with just module-info. So this skip excludes them from the aggregated javadoc.